src/main/scala/org/allenai/plugins/DeployPlugin.scala

```diff
@@ -264,30 +264,85 @@ object DeployPlugin extends AutoPlugin {
     stageDir
   }
 
-  /** Task used to deploy the current project to a remote host. */
-  lazy val deployTask = deploy := {
-    // Dependencies
-    gitRepoClean.value
+  /** Task used to load deployment configuration for a specific environment from the file
+    * conf/deploy.conf, with optional overrides. Overrides can be specified either as
+    * property-style arguments or as HOCON in ~/.deployrc.
+    *
+    * Takes as arguments:
```

Comment:
Doesn't this require the user to now type "loadDeployConfigTask prod"? Can you still type "deploy prod"?

Actually, this isn't even a setting key - will it be even visible to the user?